server/views/partials/footer.njk

```diff
@@ -1,5 +1,6 @@
 {% from "govuk/components/footer/macro.njk" import govukFooter %}
 
+<footer class="govuk-footer govuk-!-display-none-print">
```

Comment:
given what we saw in incentives-ui, this `<footer>` wrapper should be removed and you can create the no-print effect with `classes: 'govuk-!-display-none-print'` inside `govukFooter`